### PR TITLE
Default user

### PR DESCRIPTION
**Pull request by tplevko**

### CODE REVIEW

File: pom.xml
Comment (dsimansk) [inline pom.xml:20]: Is the downgrade required to be compatible with currently used version of KC?

File: pom.xml
Comment (dsimansk) [inline pom.xml]: There's another PR #1 that adds fabric8's client, I would preferred to use that one.

**Review by dsimansk — CHANGES_REQUESTED** [review]
Some comments inline, pls rebase after merging PR #1 and squash the commits :)

File: pom.xml
Comment (tplevko) [reply pom.xml:20]: yes, the newer client caused problems

File: pom.xml
Comment (tplevko) [reply pom.xml]: ok, will look into that

**Comment by tplevko** [comment]
Reflected the changes made by Kuba. Ready for second review.

File: src/test/java/io/syndesis/qe/rest/tests/AbstractSyndesisRestTest.java
Comment (dsimansk) [inline src/test/java/io/syndesis/qe/rest/tests/AbstractSyndesisRestTest.java:53]: Are you sure that it work with username+password and omitting token?

Comment (tplevko) [reply src/test/java/io/syndesis/qe/rest/tests/AbstractSyndesisRestTest.java:53]: np, tried it. Made this change, because we need the uname/pass specified for case when the OS user is not yet created (happens when we use syndesis on minishift), so the token would be redundant.

**Review by dsimansk — APPROVED** [review]

**Pull request merged**
